terraform/ec2/main.tf

```diff
@@ -210,7 +213,7 @@ resource "null_resource" "download_collector_from_local" {
 
 resource "null_resource" "download_collector_from_s3" {
   depends_on = [null_resource.check_patch]
-  count = var.install_package_source == "s3" ? 1 : 0
+  count = var.install_package_source == "s3" && !var.ssm_mode ? 1 : 0
```

Comment:
Can we reuse `install_package_source` to have `ssm` instead of having both `install_package_source` and `ssm_mode`?

Reply:
Agree.